src/phone-input/phone-input.js

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React, {useRef} from 'react';
+
+import {Input as DefaultInput} from '../input/index.js';
+import {SIZE} from './constants.js';
+import CountrySelect from './country-select.js';
+import {getOverrides, mergeOverrides} from '../helpers/overrides.js';
+
+import type {PropsT} from './types.js';
+
+export default function PhoneInput(props: PropsT) {
+  const {
+    inputValue,
+    onInputChange,
+    size = SIZE.default,
+    overrides = {
+      Input: {},
```

Comment:
I think usually we only define an empty object here https://github.com/uber-web/baseui/blob/master/src/select/select.js#L817